deploy/acm-policies/50-GENERATED-backplane-acs.Policy.yaml

```diff
@@ -0,0 +1,791 @@
+                                - servicemonitors
+                                - thanosrulers
+                              verbs:
+                                - get
+                                - list
+                                - watch
+                            - apiGroups:
+                                - observability.redhat.com
+                              resources:
+                                - observabilities
+                              verbs:
+                                - get
+                                - list
+                                - watch
+                            - apiGroups:
+                                - integreatly.org
+                              resources:
+                                - grafanas
+                                - grafanadatasources
+                                - grafanadashboards
+                              verbs:
+                                - get
+                                - list
+                                - watch
+                            - apiGroups:
+                                - ""
+                              resources:
+                                - persistentvolumeclaims
+                              verbs:
+                                - patch
+                            - apiGroups:
+                                - apps
+                              resources:
+                                - statefulsets
+                              verbs:
+                                - patch
+                    - complianceType: mustonlyhave
+                      metadataComplianceType: musthave
+                      objectDefinition:
+                        apiVersion: rbac.authorization.k8s.io/v1
+                        kind: RoleBinding
+                        metadata:
+                            name: backplane-acs-rhacs-observability
+                            namespace: rhacs-observability
+                        roleRef:
+                            apiGroup: rbac.authorization.k8s.io
+                            kind: Role
+                            name: backplane-acs-rhacs-observability
+                        subjects:
+                            - apiGroup: rbac.authorization.k8s.io
+                              kind: Group
+                              name: system:serviceaccounts:openshift-backplane-addon-acs-fleet-shard
+                pruneObjectBehavior: DeleteIfCreated
+                remediationAction: enforce
+                severity: low
+    remediationAction: enforce
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+    name: placement-backplane-acs
+    namespace: openshift-acm-policies
+spec:
+    clusterSelector:
+        matchExpressions:
+            - key: api.openshift.com/addon-acs-fleetshard
+              operator: In
+              values:
+                - "true"
```

Comment:
I guess the label to the ManagedCluster is coming from a different place? Anyway, this works.

Reply:
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/111028